integration-tests/check-subscriptions

```diff
@@ -402,6 +402,14 @@ class TestSubscriptions(SubscriptionsCase):
         m = self.machine
         b = self.browser
 
+        # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=2062136
+        #
+        # We rely on insights-client.service to be working, let's not
+        # get distracted by SELinux.  Denials will still be logged and
+        # tracked as known issues even when not enforcing.
+        #
+        m.execute("setenforce 0")
```

Comment:
Is this only temporary workaround? Will it be possible to revert, when https://bugzilla.redhat.com/show_bug.cgi?id=2062136 is properly fixed?